src/php/views/partials/content-single.twig

```diff
@@ -1,8 +1,19 @@
-<article id="post-{{ post.id }}">
-	<h2>
-		{{ post.title }}
-	</h2>
+<article id="post-{{ post.id }}" class="post-preview">
```

Comment:
Naming things is hard, I wasn't sure about `post-preview` here, if you have a better idea I am happy to hear it!

Reply:
I don't see anything else conflicting with this class name, sounds good to me! 👍